### ISSUE TITLE
AlphaImageLoader is mangled after compression

### ISSUE THREAD

**Issue by ben-eb** [jakubpawlowicz/clean-css#489]
```css
body {
    filter: progid:DXImageTransform.Microsoft.AlphaImageLoader(src='images/skyline.jpg', sizingMethod='scale');
}
```

After compression:

```css
body{filter:alphaImageLoader(src='images/skyline.jpg', sizingMethod='scale')}
```

I know it's an old property and is only relevant in IE8 and older, but I would expect that compatibility mode preserves this syntax.

**Comment by jakubpawlowicz** [comment]
It definitely should. It works fine in 2.2 but not anymore in 3.x...

**Comment by jakubpawlowicz** [comment]
Fixed in 3.1.6!

**Comment by ben-eb** [comment]
Awesome, thanks for the speedy resolution. :+1:
